modular_skyrat/modules/cargo/code/packs.dm

```diff
@@ -249,6 +249,14 @@
 					/obj/item/stack/sheet/cloth/ten,
 					/obj/item/stack/sheet/cloth/ten)
 	crate_name = "advanced art supplies"
+	
+/datum/supply_pack/misc/bicycle // overwritten
```

Comment:
As much as I'd love to sneakily overwrite this, comment it out in the main file, set this as a medical supply pack.

Reply:
all-done